content/doc/administration/requirements/java.adoc

```diff
@@ -9,11 +9,12 @@ There are separate run and job execution requirements for Jenkins installations.
 
 Modern Jenkins versions have the following Java requirements:
 
-* Java 8 is the **ONLY** currently supported runtime environment, both 32-bit and 64-bit versions are supported
+* Java 8 runtime environments, both 32-bit and 64-bit versions are supported
+* Since Jenkins `2.164` footnote:[since `2.155` but it was made easier in `2.164`], Java 11 runtime environments are supported
+** in order to run Jenkins on Java 11, there are few precautions to take, which are detailed in the link:/doc/administration/requirements/jenkins-on-java-11[Running Jenkins on Java 11] documentation.
```

Comment:
There is no precautions on the linked page, everything is documented in the Upgrade guide

```suggestion
** Running Jenkins with Java 11 is documented link:/doc/administration/requirements/jenkins-on-java-11[here] 
```